pandas/compat/pickle_compat.py

```diff
@@ -50,26 +52,31 @@ def load_reduce(self):
             print(func, args)
         raise
 
-    stack[-1] = value
```

Comment:
Because Python isn't an interpreted language :smile: (that line of code is unreachable, so we can never get an error there)?